### PR TITLE
Add contribute.json file.

### PR DESCRIPTION
**Pull request by Osmose**
Add a contribute.json file à la http://contribute.paas.allizom.org/.

### CODE REVIEW

**Comment by Osmose** [comment]
@peterbe r?

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "django-browserid",
+    "description": "django-browserid is a library that integrates BrowserID authentication into Django.",
```
Comment (bobsilverberg) [inline contribute.json]: @Osmose I know it's the name of the package, but is `BrowserID` still a valid term? Should the description use the term `Persona` instead?

Comment (peterbe) [inline contribute.json]: @bobsilverberg I think Persona is the Mozilla **instance** of the technology called "BrowserID".

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "irc-contacts": ["Osmose", "peterbe", "willkg"]
+    },
+    "bugs": {
+        "list": "https://github.com/mozilla/django-browserid/issues",
+        "report": "https://github.com/mozilla/django-browserid/issues/new"
+    },
+    "keywords": [
+        "django",
+        "python",
+        "persona"
```
Comment (peterbe) [inline contribute.json]: add `browserid`. Isn't my comment above true?

**Comment by peterbe** [comment]
r+ with nit.

File: contribute.json (hunk 1 above)
Comment (Osmose) [inline contribute.json]: Correct! The underlying protocol is called BrowserID. See http://django-browserid.readthedocs.org/en/latest/user/intro.html#persona-dependence for more info.

**Pull request merged**